unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityInformation/SmartItems/Parameters/Actions/ActionEvent.cs

```diff
@@ -0,0 +1,9 @@
+using DCL.Components;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ActionEvent 
+{
+    public List<DCLBuilderInWorldEntity> entityList;
+}
```

Comment:
Yes, to show the list of actions in `ActionsListView ` it should be wrapped in a class. Also, this class will probably be expanded in the future, right now we don't show any UI related to actions, but if we need it in the future, it will contain more properties showing the state of the Action